std_msgs/Header header
int32 fruit_id
string fruit_type
string fruit_variety
string data